packages/TorneloScoresheet/src/context/AppModeStateContext.tsx

```diff
@@ -1,48 +1,17 @@
 import React, { useContext, useState } from 'react';
-import { makegoToTablePairingSelection as makeEnterPgnToPairingSelection } from '../goToTablePairingSelection';
 import { AppModeState, AppMode } from '../types/AppModeState';
-import { GameInfo } from '../types/chessGameInfo';
-import { Result } from '../types/Result';
 
 // The global state for the app
-const AppModeStateContext = React.createContext<
+export const AppModeStateContext = React.createContext<
   [AppModeState, React.Dispatch<React.SetStateAction<AppModeState>>]
 >([{ mode: AppMode.EnterPgn }, () => undefined]);
 
-type AppModeStateHookType = [
-  AppModeState,
-  {
-    enterPgnToPairingSelection: (
-      liveLinkUrl: string,
-    ) => Promise<Result<undefined>>;
-    pairingSelectionToTablePairing: (pairing: GameInfo) => void;
-    pairingSelectionToEnterPgn: () => void;
-  },
-];
+type AppModeStateHookType = [AppModeState];
```

Comment:
No need to have this type, or make it a tuple anymore I don't think. We can just have the hook return `AppModeState` directly